app/models/post.rb

```diff
@@ -1542,6 +1553,31 @@ def as_json(options = {})
       super(options)
     end
 
+    def minimal_attributes
+      preview_dims = preview_dimensions
+      hash = {
+          'status': status,
```

Comment:
These keys are symbols, not strings. This is identical to
```rb
hash = {
  status: status,
  flags: status_flags,
  # ...
}
```

If you meant to use strings (I don't think it matters in this case), you would write this:
```rb
hash = {
  'status' => status,
  'flags' => status_flags,
  # ...
}
```

Reply:
Augh, you're correct. I was thinking of making them into strings, but this is indeed the symbol syntax. I'll just make them proper symbols and be done with it.